addr_test.go

```diff
@@ -0,0 +1,27 @@
+package reuseport
+
+import (
+	"testing"
+)
+
+type netAddr struct {
+	address string
+	network string
+}
+
+func TestResolveAddr(t *testing.T) {
```

Comment:
Why do you think it's not useful? I added the test because `ResolveAddr` is an exported method of this package and the test verifies the method will resolve addresses for these protocols. While the test may seem silly given that we can see the implementation, its purpose is to verify behaviour offered by `ResolveAddr` from a user's perspective. So if someone were to refactor `ResolveAddr` the test would help them ensure that no functionality was lost.